contracts/AssetToken.sol

```diff
@@ -86,6 +106,27 @@ contract AssetToken is ERC223Interface, IERC20 {
         return _balances[who];
     }
 
+
+    function setEmergencyPermission(address who) public onlyOwner {
+        _emergencyDelegate = who;
+        emit EmergencyDelegation(_emergencyDelegate);
+    }
+
+    function revokeEmergencyPermission() public onlyOwner {
```

Comment:
You can do  this operation by passing the owners address into `setEmergencyPermission`